### PR TITLE
Gen3 Monthly Release 2022.11 QA-BRH 2022.11 qa-brh.planx-pla.net 1670347487

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2022.11 to qa-brh.planx-pla.net

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## Deployment changes
- fence
  - Configure the `ALLOWED_DATA_UPLOAD_BUCKETS` setting to allow users to upload to buckets other than `DATA_UPLOAD_BUCKET` (https://github.com/uc-cdis/fence/pull/1051)
  - B/c the default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP, you MUST ensure that an instance that should comply with FedRAMP does NOT override this fence configuration (e.g. it should be 15 minutes) `SESSION_TIMEOUT: 900` (https://github.com/uc-cdis/fence/pull/1040)
## Breaking changes
- arborist
  - The `GET /auth/mapping` endpoint does not accept `username` as a query parameter anymore. It only supports parsing the username from the provided JWT. To specify a username, use the `POST /auth/mapping` endpoint. (https://github.com/uc-cdis/arborist/pull/153)
- fence
  - Default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP (https://github.com/uc-cdis/fence/pull/1040)
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (https://github.com/uc-cdis/indexd/pull/336)

**Review by aartivnkt — DISMISSED** [review]
Update to latest versions of microservices

File: qa-brh.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -8,23 +8,23 @@
...
-    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2.0.1",
-    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2022.09",
-    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2022.09",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:5.0.0",
-    "requestor": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/requestor:1.7.1",
-    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.09",
-    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.09",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:0.4.1"
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2022.11",
+    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2022.11",
```
Comment (tianj7) [inline qa-brh.planx-pla.net/manifest.json:20]: Can we change the mds version to 2.1.0 to include the newest ICDC adapter for Agg mds?
https://github.com/uc-cdis/metadata-service/releases/tag/2.1.0

**Review by tianj7 — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## Deployment changes
- fence
  - Configure the `ALLOWED_DATA_UPLOAD_BUCKETS` setting to allow users to upload to buckets other than `DATA_UPLOAD_BUCKET` (https://github.com/uc-cdis/fence/pull/1051)
  - B/c the default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP, you MUST ensure that an instance that should comply with FedRAMP does NOT override this fence configuration (e.g. it should be 15 minutes) `SESSION_TIMEOUT: 900` (https://github.com/uc-cdis/fence/pull/1040)
## Breaking changes
- arborist
  - The `GET /auth/mapping` endpoint does not accept `username` as a query parameter anymore. It only supports parsing the username from the provided JWT. To specify a username, use the `POST /auth/mapping` endpoint. (https://github.com/uc-cdis/arborist/pull/153)
- fence
  - Default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP (https://github.com/uc-cdis/fence/pull/1040)
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (https://github.com/uc-cdis/indexd/pull/336)

**Pull request merged**